src/pyscaffold/cli.py

```diff
@@ -139,10 +139,30 @@ def parse_args(args):
 
     # Parse options and transform argparse Namespace object into common dict
     opts = vars(parser.parse_args(args))
+    return opts
+
+
+def process_opts(opts):
+    """Process and enrich command line arguments
+
+    Args:
+        opts (dict): dictionary of parameters
+
+    Returns:
+        dict: dictionary of parameters from command line arguments
+    """
+    # When pretending the user surely wants to see the output
+    if opts['pretend']:
+        opts['log_level'] = logging.INFO
 
     # In case of an update read and parse setup.cfg
     if opts['update']:
-        opts = info.project(opts)
+        try:
+            opts = info.project(opts)
+        except Exception as e:
+            if opts.get('log_level', logging.ERROR) <= logging.INFO:
```

Comment:
I think that `logger.error(..., exc_info=True)` would avoid the conditional...
But I understand that at this point we are postponing the use of `pyscaffold.log.logger` because it is just fully configured in the `pyscaffold.api.create_project` function.

Do you think it is worthy to review this design decision?

Reply:
Hmm.. good point. Have you an idea for a better resolution like moving the initialisation of the logger more upstream into cli.py? Would we lose something by doing so?

Reply:
@abravalheri I moved the initialization of the logger to `cli.py` since I guess it also makes more sense to have it there. Let me know if you are fine with this and if we can merge this PR.